pkg/plugin/test-keeper/plugin_hint_file_matcher_test.go

```diff
@@ -0,0 +1,81 @@
+package testkeeper_test
+
+import (
+	"regexp"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/plugin/test-keeper"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test keeper comment message parsing", func() {
+
+	Context("Plugin hint file matcher", func() {
+
+		It("should match plugin hint file from relative path", func() {
+			// given
+			pluginHint := "path/to/test-keeper_hint.md"
+
+			// when
+			isFilePath, _ := regexp.MatchString(testkeeper.FileRegex, pluginHint)
```

Comment:
@bartoszmajsak, aren't these- https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/test-keeper/comment_message_test.go sufficient tests for this functionality?

Reply:
I understand this as a test that the regexp is correct - for this purpose, this test is fine I think (but not sure if really needed). It can be also verified one level higher - in the `comment_message_test.go` - for example, the case with an inline message is missing there.

Reply:
@MatousJobanek, here is the test for the inline message - https://github.com/arquillian/ike-prow-plugins/pull/84/files#diff-f47424f71ebd94efde33032a25a6905aR198

Reply:
I missed that, sorry and thx for the link :-)

Reply:
My point was more that instead of testing an actual functionality - e.g. does the specified file matches a pattern so we load its content instead of treating it as an inline comment we simply have tests exercising `regexp.MatchString` which is an implementation detail. This is zero test coverage test also :)

Another thing is that we treat any file suffixed with `_hint.md` as an actual file to load content from, rather than saying "if the file is named after pluginname and `_hint.md` suffix we treat it as file, otherwise it is inline comment.

I don't want to keep this PR open for ages, so I am ok with the latter problem. But the test as such doesn't really bring much value as it's disconnected from the actual functionality. Also notice that `.FileRegex` is made public for the sake of the existence of this test.

> @bartoszmajsak, aren't these- https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/test-keeper/comment_message_test.go sufficient tests for this functionality?

As a matter of fact this test should fail now https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/test-keeper/comment_message_test.go#L86, at least based on what is stated in the docs.